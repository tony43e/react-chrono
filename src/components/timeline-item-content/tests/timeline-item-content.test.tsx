import { render, screen } from "@testing-library/react";
import React from "react";
import Content from "../timeline-item-content";

test("Test Timeline Item Content", () => {
  const { getByText } = render(<Content content="This is a test" />);
  const contentElement = getByText(/test/i);
  expect(contentElement).toBeInTheDocument();
});

test("Test Timeline Item Content Snapshot", () => {
  const { getByText } = render(<Content content="This is a test" />);
  const contentElement = getByText(/test/i);
  expect(contentElement).toMatchSnapshot();
});

test("Test Timeline Item Content Snapshot", () => {
  render(<Content content="This is a test" active />);
  const element = screen.getByText(/this is a test/i);
  expect(element.parentElement).toHaveClass("active");
  expect(element.parentElement).toHaveStyle("color: #0f52ba;");
});
