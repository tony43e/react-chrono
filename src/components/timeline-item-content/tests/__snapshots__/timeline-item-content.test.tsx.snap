// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Timeline Item Content Snapshot 1`] = `
<span
  class="sc-AxirZ izgUJV"
>
  This is a test
</span>
`;
